include/solvers/adaptive_time_solver.h

```diff
@@ -82,7 +82,9 @@ class AdaptiveTimeSolver : public FirstOrderUnsteadySolver
 
   virtual void integrate_adjoint_sensitivity(const QoISet & qois, const ParameterVector & parameter_vector, SensitivityData & sensitivities) override = 0;
 
-  virtual Real last_complete_deltat() override { return completedtimestep_deltat; }
+  virtual void integrate_adjoint_refinement_error_estimate(AdjointRefinementEstimator & adjoint_refinement_error_estimator, ErrorVector & QoI_elementwise_error, std::vector<Real *> QoI_time_instant = std::vector<Real *>()) = 0;
+
+  virtual Real last_complete_deltat() { return completedtimestep_deltat; };
```

Comment:
Why'd you take off ```override``` here?

Reply:
Sorry, this was probably mess up due to a conflict. I had been futzing with this function and had thought I had restricted it to AdaptiveTS and subclasses only. I'll correct this.

Reply:
Fix commit: